public/video-ui/src/components/Video/Display.js

```diff
@@ -28,14 +29,38 @@ class VideoDisplay extends React.Component {
     }
   }
 
+  canonicalVideoPageExists = () => {
+    return this.props.usages.totalVideoPages > 0;
+  };
+
+  getComposerUrl = () => {
+    return getStore().getState().config.composerUrl;
+  };
+
   saveAndUpdateVideo = video => {
     if (this.props.route.mode === 'create') {
       this.props.videoActions.createVideo(video)
         .then(() => {
           this.props.videoActions.getUsages(this.props.video.id);
         });
     } else {
-      this.props.videoActions.saveVideo(video);
+      this.props.videoActions.saveVideo(video)
+      .then(() => {
+        if (this.canonicalVideoPageExists()) {
+          const videoBlock = getVideoBlock(
```

Comment:
Good point, we probably shouldn't because otherwise publishing an atom doesn't make much sense. I'll need to update my composer endpoint pr to support this.